cmd/ak/cmd/sessions/delete.go

```diff
@@ -0,0 +1,58 @@
+package sessions
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete [session ID] [--fail]",
+	Short:   "Delete non-running session",
+	Aliases: []string{"d"},
+	Args:    cobra.MaximumNArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			id, err := latestSessionID()
+			if err != nil {
+				return err
+			}
+			args = append(args, id)
+		}
+
+		r := resolver.Resolver{Client: common.Client()}
+		s, id, err := r.SessionID(args[0])
+		if err != nil {
+			return err
+		}
+
+		if s != nil {
+			ctx, cancel := common.LimitedContext()
+			defer cancel()
+
+			common.RenderKVIfV("session", s) // FIXME: should we print deleted session?
```

Comment:
If the user doesn't have to specify the sessions ID, for sure yes, but at the end, not here. If we decide to require specifying the ID (see my other comment above) it's preferable to adhere to the "silence == success" rule.

Reply:
I feel that in case of delete CMD the silence rule isn't good enough, but the opposite. This gives the user immediate opportunity to see what was deleted and act. So if CLI is invoked by the user itself, I think we should provide clear output what was deleted. If CLI is scripted the output doesn't matter and there is --fail option that should cover the cases needed for scripting

Reply:
@daabr lots of CLIs has a --quiet option to suppress any output
maybe that's a win win options if we set the output to quiet as default or something like that

regarding delete, I tend to agree with @ig-ra 
I prefer to know what happened instead of just empty line